weblate/trans/formats.py

```diff
@@ -809,6 +824,21 @@ class PropertiesUtf8Format(FileFormat):
     loader = ('properties', 'javautf8file')
     monolingual = True
 
+    @classmethod
+    def supports_new_language(cls):
+        '''
+        Checks whether we can create new language file.
+        '''
+        return True
+
+    @staticmethod
+    def add_language(filename, code, base):
+        '''
+        Adds new language file.
+        '''
+        with open(filename, 'w') as output:
+            output.write('\n')
+
```

Comment:
I think it's better idea to share the code, so can you please rebase your changes on current master where I've changed the properties classes to inherit?